### PR TITLE
Document symfony mailer

### PR DESCRIPTION
**Pull request by miaulalala**
Fixes https://github.com/nextcloud/documentation/issues/9600

| Before | After |
|--------|--------|
| screencapture-127-0-0-1-8000-configuration-server-email-configuration-html-2023-04-06-21_42_23 |screencapture-127-0-0-1-8000-configuration-server-email-configuration-html-2023-04-12-12_56_43 |

### CODE REVIEW

**Review by nickvergessen — APPROVED** [review]

File: admin_manual/configuration_server/email_configuration.rst [hunk 1]
```diff
@@ -35,22 +35,28 @@ Configuring an SMTP server
...
-* Authentication method: None, Login, Plain, or NT LAN Manager
+* Authentication method: always 'Login' - this option will only apply when authentication is required
 
 * The server's IP address or fully-qualified domain name and the SMTP port
 
 * Login credentials (if required)
 
 .. note:: The ``overwrite.cli.url`` parameter from ``config.php`` will be used for the SMTP EHLO.
 
-.. figure:: ../images/smtp-config-smtp.png
```
Comment (nickvergessen) [inline admin_manual/configuration_server/email_configuration.rst:52]: delete this file, or is it still referenced from somewhere?

**Review by st3iny — COMMENTED** [review]
The section regarding manual TLS configuration is not relevant anymore and can be removed. AFAIK, STARTTLS is used automatically and can't be enforced (as stated in the beginning of the document).

`Setting mail server parameters in config.php` -> `SMTP` -> `TLS`

grafik

**Review by st3iny — APPROVED** [review]
I merged the SSL and SMTP sections because it doesn't make sense to have it separate anymore. I also added the paragraph about authentication, which I recommended to delete earlier :see_no_evil: 

grafik

**Pull request merged**